### PR TITLE
Add @af-utils virtual and table package definitions

### PR DESCRIPTION
**Pull request by huner2**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## af-utils__react-table ([<kbd>unpkg</kbd>](https://unpkg.com/browse/af-utils__react-table@latest/))
was missing the following properties:
1. ColumnModel
1. DefaultTableComponents
## af-utils__react-virtual-headless ([<kbd>unpkg</kbd>](https://unpkg.com/browse/af-utils__react-virtual-headless@latest/))
was missing the following properties:
1. EVT_ALL
1. EVT_FROM
1. EVT_SCROLL_SIZE
1. EVT_SIZES
1. EVT_TO
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 911e2c7e2f8c96d20791e66751a48761d34f5558
</p>

File: types/af-utils__react-table/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,153 @@
+// Type definitions for @af-utils/react-table 0.0
+// Project: https://github.com/nowaalex/af-utils
+// Definitions by: Hunter Ross <https://github.com/huner2>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as React from 'react';
+import * as Headless from 'af-utils__react-virtual-headless';
```
Comment (peterblazejewicz) [inline types/af-utils__react-table/index.d.ts:7]:
```suggestion
import * as Headless from '@af-utils/react-virtual-headless';
```
you would have eventually configure path in config file

Comment (peterblazejewicz) [inline types/af-utils__react-table/index.d.ts:7]: Should that module be also re-exported?
https://github.com/nowaalex/af-utils/blob/master/packages/af-utils-react-table/src/index.js#L4

File: types/af-utils__react-virtual-headless/af-utils__react-virtual-headless-tests.tsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
+import {
+    useVirtual,
+    useVirtualModel,
+    useSubscription,
+    EVT_FROM,
+    useComponentSubscription,
+    useOnce,
+    Subscription,
+    mapVisibleRange,
+} from 'af-utils__react-virtual-headless';
```
Comment (peterblazejewicz) [inline types/af-utils__react-virtual-headless/af-utils__react-virtual-headless-tests.tsx:10]:
```suggestion
} from '@af-utils/react-virtual-headless';
```

File: types/af-utils__react-table/af-utils__react-table-tests.tsx [hunk 3]
```diff
@@ -0,0 +1,113 @@
+import * as React from 'react';
+import * as Table from 'af-utils__react-table';
```
Comment (peterblazejewicz) [inline types/af-utils__react-table/af-utils__react-table-tests.tsx:2]:
```suggestion
import * as Table from '@af-utils/react-table';
```

File: types/af-utils__react-virtual-list/af-utils__react-virtual-list-tests.tsx [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import { List } from 'af-utils__react-virtual-list';
```
Comment (peterblazejewicz) [inline types/af-utils__react-virtual-list/af-utils__react-virtual-list-tests.tsx:1]:
```suggestion
import { List } from '@af-utils/react-virtual-list';
```

File: types/af-utils__react-table/index.d.ts (hunk 1 above)
Comment (huner2) [reply types/af-utils__react-table/index.d.ts]: I wasn't 100% sure how to handle that, I've now re-exported in both table and list.

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@huner2 thanks!

**Comment by huner2** [comment]
Ready to merge

**Pull request merged**